src/fastoad/notebooks/02_explanation/01_summary.ipynb

```diff
@@ -0,0 +1,229 @@
+{
```

Comment:
<p>About the plots: the Aerodynamic<strong>s </strong>(please fix also in source files) component should not receive MTOW, should it ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Actually, from the models I decided to use it does. The aerodynamic compute the cruise lift-to-drag ratio based on a simple quadratic model for the drag which uses a lift coefficient computed based on the lift equation with the MTOW.

I see two solutions if that's too pessimistic of an assumption : 
- Take a mass equal to a ratio of the MTOW 
- Compute an average mass between MTOW and OWE+payload but that would change the architecture of the problem by putting aerodynamics after weight computation, which changes nothing from a physical point of view but would make me change a lot of files and graphs (which I'm not too thrilled about but would understand the need to change ^^')

Reply:
<p>No, I'm fine with that. Actually, I even don't understand why I asked this question in the first place, since the purpose is clear enough in the code.</p><p> </p><p>Yet, I still think there should be a "s" at the end of "aerodynamic<strong>s</strong>" (which should apply also to file and folder names) 😅</p>

Reply:
Done in 349d2785